.github/workflows/deploy.yml

```diff
@@ -0,0 +1,121 @@
+name: deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - lab
+      - master
+      - published
+
+# these permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: Prepare
+        run: |
+          JEKYLL_ENV=development
+          DOCS_AWS_REGION=us-east-1
+          if [ "${{ github.ref }}" = "refs/heads/master" ]; then
+            DOCS_URL="https://docs-stage2.docker.com"
+            DOCS_AWS_IAM_ROLE="arn:aws:iam::710015040892:role/stage-docs-docker.github.io-20220816140248629900000003"
+            DOCS_S3_BUCKET="stage-docs-docker.github.io"
+            DOCS_S3_CONFIG="s3-config.json"
+            DOCS_CLOUDFRONT_ID="E1R7CSW3F0X4H8"
+            DOCS_LAMBDA_FUNCTION_REDIRECTS="DockerDocsRedirectFunction-stage"
+            DOCS_SLACK_MSG="Successfully deployed docs-stage2 from master branch. https://docs-stage2.docker.com/"
+          elif [ "${{ github.ref }}" = "refs/heads/published" ]; then
+            JEKYLL_ENV=production
+            DOCS_URL="https://docs2.docker.com"
```

Comment:
Question; do we have something in place to disable indexing on these temporary domains? As this is using the `production` config, it may have indexing / google analytics /etc. enabled?

Reply:
Good point, I think we can disable production for this temporary environment for now.